add_me_to_your_PATH/sysimage.jl

```diff
@@ -34,5 +18,4 @@ function main()
     create_sysimage(packages; replace_default=true, cpu_target = "generic;sandybridge,-xsaveopt,clone_all;haswell,-rdrnd,base(1)")
 end
 
-main()
-Pkg.rm("PackageCompiler")
+@time main()
```

Comment:
is `@time` necessary here?  timing information will be reported by the docker build tool for this step right?